src/OpenTelemetry.Instrumentation.SqlClient/SqlClientInstrumentation.cs

```diff
@@ -25,6 +26,19 @@ internal sealed class SqlClientInstrumentation : IDisposable
     {
         internal const string SqlClientDiagnosticListenerName = "SqlClientDiagnosticListener";
 
+        private static readonly HashSet<string> DiagnosticSourceEvents = new()
```

Comment:
nit: declare inside #if `!NETFRAMEWORK`

Reply:
Done. Thank you!

Reply:
need to wrap the using as well separately inside #if `!NETFRAMEWORK`.  Like [this](https://github.com/open-telemetry/opentelemetry-dotnet/blob/804c448d2e5a8d53405624aecd4f315261a2f379/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs#L22-L24)

Reply:
Oh, got it. Appreciate for your patient!